packages/ui/src/working-groups/modals/IncreaseWorkerStakeModal/IncreaseWorkerStakeModal.tsx

```diff
@@ -26,7 +27,12 @@ export interface IncreaseStakeFormFields {
 
 const StakeFormSchema = Yup.object().shape({
   stake: BNSchema.test(minContext('You need at least ${min} stake', 'minAddStake'))
-    .test(maxContext('Given amount exceed your transferable balance of ${max} tJOY', 'totalBalance'))
+    .test(
+      maxContext(
+        'Given amount exceed your transferable balance of' + '${max}' + `'${CurrencyName.integerValue}'`,
```

Comment:
This should work:
```suggestion
        `Given amount exceed your transferable balance of \${max} ${CurrencyName.integerValue}`,
```
If not you could pass: `Given amount exceed your transferable balance of ${'${max}'} ${CurrencyName.integerValue}`